src/System.Xml.XmlSerializer/pkg/System.Xml.XmlSerializer.pkgproj

```diff
@@ -1,13 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <Version>4.0.12.0</Version>
+    <Version>4.1.0.0</Version>
   </PropertyGroup>
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
-    <ProjectReference Include="..\ref\4.0.0\System.Xml.XmlSerializer.depproj">
-      <SupportedFramework>net45;netcore45;wp8;wpa81</SupportedFramework>
-    </ProjectReference>
     <ProjectReference Include="..\ref\System.Xml.XmlSerializer.csproj">
       <SupportedFramework>net46;netcore50;netcoreapp1.0;$(AllXamarinFrameworks)</SupportedFramework>
```

Comment:
You are going to have to reduce this supported framework list down to netcoreapp1.1 and possibly net463. That is the netcore50 error you are seeing as that doesn't support this version of the contract. See PR https://github.com/dotnet/corefx/pull/11226 as an example.